backend/controllers/emailDomainValidation.ts

```diff
@@ -1,5 +1,5 @@
 const isValidDomainNameChars = (domainName: string) => domainName.match(/^[0-9A-Za-z-,.()'&]*$/)
-const isValidDomainDescriptionChars = (domainDesc: string) => domainDesc.match(/^[0-9A-Z_]*$/)
+const isValidDomainDescriptionChars = (domainDesc: string) => domainDesc.match(/^[0-9A-Z_\s+]*$/)
```

Comment:
I'm not convinced these are correct - do we really allow (), etc in email domains but not the description?  seems odd.  This is possibly a bug from previous work so may need a separate PR to investigate

Reply:
Is this new validation regex ?

Reply:
That is out of scope of this ticket. But, its a quick regex update when the requirement is confirmed. Thomas out BA can be involved in that.

Reply:
It was suggested validation should be removed to match existing basic empty string and string length check.